### PR TITLE
Makefile: installation of conf.sh and LICENSE added

### PR DESCRIPTION
**Pull request by Vouivre**
Solve https://github.com/pystardust/ytfzf/issues/446

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,13 +1,16 @@
...
 MANDIR=${PREFIX}/share/man
 
-man:
+doc:
 	mkdir -p ${DESTDIR}${MANDIR}/man1
 	mkdir -p ${DESTDIR}${MANDIR}/man5
+	mkdir -p ${DESTDIR}${DOCDIR}
 	install docs/man/ytfzf.1 ${DESTDIR}${MANDIR}/man1
 	install docs/man/ytfzf.5 ${DESTDIR}${MANDIR}/man5
+	install docs/conf.sh LICENSE ${DESTDIR}${DOCDIR}
```
Comment (Euro20179) [inline Makefile:13]: I should've mentioned this earlier, but `LICENSE` should go in `${PREFIX}/share/licenses/ytfzf`

File: Makefile [hunk 2]
```diff
@@ -17,6 +22,7 @@ install:
 uninstall:
 	rm -f ${DESTDIR}${MANDIR}/man1/ytfzf.1
 	rm -f ${DESTDIR}${MANDIR}/man1/ytfzf.5
+	rm -f ${DESTDIR}${DOCDIR}
 	rm -f ${DESTDIR}${BINDIR}/${PROG}
 
 .PHONY: install uninstall man
```
Comment (Euro20179) [inline Makefile:28]: forgot to change `man` to `doc`

File: Makefile [hunk 3]
```diff
@@ -1,13 +1,18 @@
...
-man:
+doc:
 	mkdir -p ${DESTDIR}${MANDIR}/man1
 	mkdir -p ${DESTDIR}${MANDIR}/man5
+	mkdir -p ${DESTDIR}${DOCDIR}
+	mkdir -p ${DESTDIR}/share/licenses
 	install docs/man/ytfzf.1 ${DESTDIR}${MANDIR}/man1
 	install docs/man/ytfzf.5 ${DESTDIR}${MANDIR}/man5
+	install docs/conf.sh ${DESTDIR}${DOCDIR}
+	install LICENSE ${DESTDIR}/share/licenses
```
Comment (Euro20179) [inline Makefile:15]: I think this is meant to be `${DESTDIR}${PREFIX}/share/licenses`, currently it installs to `/share/licenses`

**Comment by Vouivre** [comment]
Sorry about all those commits.....

**Comment by Euro20179** [comment]
> Sorry about all those commits.....

don't worry i've done some horrible commits myself.

**Comment by Vouivre** [comment]
> don't worry i've done some horrible commits myself.

Thanks!

**Comment by Euro20179** [comment]
I changed some things, mainly I installed everything with `-Dm644` so it's not executable, I also put `LICENSE` in the right place.

**Comment by Vouivre** [comment]
I didn't know that you can modify a PR of a user. Good to know!

Sorry, for `LICENSE`. Yes, I forgot `ytfzf` at the end.

**Comment by Euro20179** [comment]
> Sorry, for LICENSE. Yes, I forgot ytfzf at the end.

All good

> I didn't know that you can modify a PR of a user. Good to know!

Yeah, you clone their repository, (in this case `https://github.com/Vouivre/ytfzf`), modifiy it, then push back to their repo.

**Pull request merged**